CarService/CarService.RepariOrders.Api/Startup.cs

```diff
@@ -25,7 +27,7 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddDapperForPostgreSQL();
             services.AddScoped<IRepairOrderRepository, RepairOrderRepository>();
-            services.AddScoped<IRepairOrdersService, RepairOrdersService>();
+            services.AddScoped<AppCore.Interfaces.IRepairOrdersService, CarService.RepariOrders.Api.Services.RepairOrdersService>();
```

Comment:
Do we need it?

Reply:
As I said above it's necessary due to name conflicts. It will be fixed with architecture cleaning in future.